jeeves/dev.py

```diff
@@ -8,6 +8,7 @@ class EnvVar(enum.StrEnum):
     DATAFOLD_K8S_SECRETFILE = "DATAFOLD_K8S_SECRETFILE"
     DATAFOLD_DEPLOY_NAME = "DATAFOLD_DEPLOY_NAME"
     DATAFOLD_DEV_OVERRIDES = "DATAFOLD_DEV_OVERRIDES"
+    DATAFOLD_DB_HOSTPATH = "DATAFOLD_DB_HOSTPATH"
```

Comment:
Ik had deze juist uit de dev gehaald, omdat deze niet gebruikt wordt verder. In ee nandere PR wordt deze juist eruit gehaald.  Op die manier lekken we geen dev details naar een prod deployment.